commandline/src/com/thoughtworks/go/util/command/CommandLine.java

```diff
@@ -1,74 +1,23 @@
-/*************************GO-LICENSE-START*********************************
+/*
  * Copyright 2016 ThoughtWorks, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *************************GO-LICENSE-END***********************************/
-
-/*
```

Comment:
Can you bring back this license? I believe this file started off as a copy from [this file](https://svn.apache.org/repos/asf/ant/core/trunk/src/main/org/apache/tools/ant/types/Commandline.java) and so, it needs to have that license as well.

This was probably waay before either of us joined. :)